anthos-bm-gcp-bash/docs/admin.md

```diff
@@ -0,0 +1,119 @@
+## Install an Anthos on bare metal cluster of type 'admin'
+
+This sample shows you how to setup an **admin** Anthos clusters on bare metal
+in High Availability (HA) mode using Compute Engine Virtual Machines (VMs). The
+[install_admin_cluster](/anthos-bm-gcp-bash/install_admin_cluster.sh) script
+encapsulates all the steps required to setup the Compute Engine VMs and to
+trigger the installation of the admin cluster. The steps used for setting up the
+Compute Engine VM environment in this script are the same as the ones explained
+in the [Try Anthos clusters on bare metal on Compute Engine VMs](https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/try/gce-vms) guide. The only differences are the number
+of Compute Engine VMs used and the cluster configuration file
+_(which is specific to an admin cluster)_.
+
+### Prerequisites
+
+- A workstation with access to the Internet _(i.e. Google Cloud APIs)_ with the following installed
+  - [Git](https://git-scm.com/)
+  - [Google Cloud SDK (gcloud CLI)](https://cloud.google.com/sdk/docs/install)
+- A [Google Cloud Project](https://console.cloud.google.com/cloud-resource-manager?_ga=2.187862184.1029435410.1614837439-1338907320.1614299892) _(in which the resources for the setup will be provisioned)_
+- The gcloud CLI must be [authenticated to Google Cloud and be configured to use
+  the Google Cloud Project](https://cloud.google.com/sdk/gcloud/reference/init) you intend to use
+---
+### Quickstart
+
+1. Clone this repo into the workstation from where the rest of this guide will
+   be followed.
+
+    ```sh
+    git clone https://github.com/GoogleCloudPlatform/anthos-samples
+    cd anthos-bm-gcp-bash
```

Comment:
do you have to cd into anthos samples too?

Reply:
Yes. Thank you 🐐 FIxed in 9942d2a7a02ec3351341f69fb8abeaee4f3f24c1